systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -619,8 +620,12 @@ protected void doKafkaConnectAndKafkaConnectorUpgradeOrDowngradeProcedure(final
 
         // Verify that Kafka cluster RU
         waitForKafkaClusterRollingUpdate();
-        // Verify that KafkaConnect pods are rolling and KafkaConnector is ready
-        RollingUpdateUtils.waitTillComponentHasRolled(testStorage.getNamespaceName(), connectLabelSelector, 1, connectPods);
+
+        // Check KafkaConnect RU only if we don't use pre-built image
+        if (Environment.CONNECT_IMAGE_WITH_FILE_SINK_PLUGIN.isEmpty()) {
+            // Verify that KafkaConnect pods are rolling and KafkaConnector is ready
+            RollingUpdateUtils.waitTillComponentHasRolled(testStorage.getNamespaceName(), connectLabelSelector, 1, connectPods);
+        }
```

Comment:
I think that this is not desired to have it here TBH, as it looks to me like a place that could hide an issue.
You can easily forget about something like this in the pipelines and skip the check -> and in case that there will be issue with RU in Connect, we will not find it soon.
In most of our CIs we are building the Connect image to speed up the build and we are skipping it for upgrade. 
You should do it for upgrade in TF as well.

Reply:
@see-quick @kornys thoughts? When is `CONNECT_IMAGE_WITH_FILE_SINK_PLUGIN` is set, the test will always fail without this change. Option would be remove the env from this pipeline and add annotation that it is unsupported on our Kind IPv6 config.

Reply:
I am more inclined to set it as un-supported on our kind IPv6 as you have suggested :).